scratches/botTest.py

```diff
@@ -0,0 +1,133 @@
+import telebot  # importing Telegram API classes and functions
+import libraries.qr_make as qr  # import qr_make lib with qr functions
+from telebot import types
+import os
+import libraries.url as val
+
+# defining global vars
+TOKEN = None
+
+with open('token.txt') as f:
```

Comment:
I think it is a good idea to refactor this into class. Global variables have side effects, Works for now, but later can shoot you in the foot.